src/commands/math/commands.js

```diff
@@ -437,7 +455,7 @@ CharCmds['/'] = P(Fraction, function(_, super_) {
 
 var SquareRoot =
 LatexCmds.sqrt =
-LatexCmds['√'] = P(MathCommand, function(_, super_) {
+LatexCmds['v'] = P(MathCommand, function(_, super_) {
```

Comment:
Replacing the unicode square root symbol with a v also seems like it might be a mistake. Also happened in https://github.com/sclower/mathquill/commit/f53cfb10caacbc1f9bf3b1d31f5589cabb78a199